napari_skimage_regionprops/_table.py

```diff
@@ -29,10 +29,17 @@ def __init__(self, layer: "napari.layers.Layer", viewer: "napari.Viewer" = None)
 
         self._view = QTableWidget()
         self._view.setEditTriggers(QTableWidget.EditTrigger.NoEditTriggers)
+
         if hasattr(layer, "properties"):
-            self.set_content(layer.properties)
-        else:
-            self.set_content({})
+            content = layer.properties
+        elif hasattr(layer, "features"):
+            content = layer.features.to_dict('list')
+
+        # to accomodate passing features to surfaces through the metadata
+        elif 'features' in layer.metadata.keys():
+            layer.features = layer.metadata['features']
+            content = layer.features.to_dict('list')
+        self.set_content(content)
```

Comment:
```suggestion
        else:
             content = {}
        self.set_content(content)
```
I _think_ we need an else-block here to not break backwards-compatibility.

Reply:
Hm...not sure on this one. Wouldn't the `layer.features` property always be set to `{}` by default at the time of creating the layer (except for the Surface layer, that is)?

Upon writing this I'm realizing that you are absolutely right. Thanks for spotting!